src/common/MouseeventManager.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { Store } from 'redux';
+import BaseManager from './BaseManager';
+import MouseeventListener from './MouseeventListener';
+
+export type Options = {
+    referenceEl: HTMLElement;
+};
+
+export type Props = {
+    store: Store;
+};
+
+export default class MouseeventManager extends BaseManager {
```

Comment:
Do we need a whole manager for this or can we follow the HighlightListener convention?

Reply:
I tried to go with the `HighlightListener` convention first, but it kind of needs the `BaseManager` logic of inserting a div to mount the react component so I ended up duplicating most of the `BaseManager`